README.md

```diff
@@ -190,6 +190,9 @@ A certificate spec has the following fields:
 * `take_actions_only_if_running`: boolean, if true, only fire a spec's action if the service is actually running.
   If this is set to false (the default for historical reasons), this can lead to certmgr starting a downed service
   when PKI expiry occurs.
+* `key_usages`: optional: An array of strings defining what this key should be used for. Certmgr will consider a cert invalid
```

Comment:
Can you link to a specific tag, rather than master?

Reply:
Resolved by linking to the 1.3.3 tag which is close to what we have vendored. I also considered linking directly to our vendored copy which would be the most accurate, but would then need updated whenever we update deps, so I think just linking to the upstream cfssl with a tagged version is best.  Luckily this map is unlikely to change.